app/src/main/java/com/varun/gbu_timetables/FavouritesFragment.java

```diff
@@ -97,10 +112,13 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             @Override
             public void onItemClick(AdapterView<?> parent, View view, int position,
                                     long id) {
-                if (ElementsCount == 0)

```

Comment:
I will have to examine this change.

Reply:
ElementsCount = 0 when no favourites added and hence disables the onitemclick(), but now onItemClick is also shared by MyClass. So if no favourite exists user cant click on MyClass too. Substitute of this statement exists at Line119